lib/notify.js

```diff
@@ -2,39 +2,44 @@ var Handlebars    = require('handlebars')
 var request       = require('request')
 var Logger        = require('./logger')
 
-module.exports.notify = function(metadata, config) {
-
-  if (config.notify && config.notify.content) {
+var handleNotify = function(metadata, notifyEntry) {
+  var regex
+  if(metadata.name && notifyEntry.packagePattern) {
```

Comment:
@juanpicado Thanks a lot, sorry for the delay on this. Migrated these changes to ES6 syntax and fixed those code styling issues.